net/socket/tcp_socket_posix.cc

```diff
@@ -237,8 +239,12 @@ int TCPSocketPosix::Bind(const IPEndPoint& address) {
   SockaddrStorage storage;
   if (!address.ToSockAddr(storage.addr, &storage.addr_len))
     return ERR_ADDRESS_INVALID;
-
-  return socket_->Bind(storage);
+  int rv = socket_->Bind(storage);
+  if (rv != 0){
+    DLOG(ERROR) << "TCPSocketPosix::Bind returned an error" << rv;
+    return MapSystemError(errno);
```

Comment:
Why do we need to make this translation call? Shouldn't `Bind` just return the right type/value?

Reply:
You are right, it should be removed. And it is removed.